heft-plugins/heft-storybook-plugin/src/StorybookPlugin.ts

```diff
@@ -55,25 +57,49 @@ export interface IStorybookPluginOptions {
   storykitPackageName: string;
 
   /**
-   * The module entry point that Heft should use to launch the Storybook toolchain.  Typically it
-   * is the path loaded the `start-storybook` shell script.
+   * The module entry point that Heft serve mode should use to launch the Storybook toolchain.
+   * Typically it is the path loaded the `start-storybook` shell script.
    *
    * @example
    * If you are using `@storybook/react`, then the startup path would be:
    *
    * `"startupModulePath": "@storybook/react/bin/index.js"`
    */
-  startupModulePath: string;
+  startupModulePath?: string;
+
+  /**
+   * The module entry point that Heft non-serve mode should use to launch the Storybook toolchain.
+   * Typically it is the path loaded the `build-storybook` shell script.
+   *
+   * @example
+   * If you are using `@storybook/react`, then the static build path would be:
+   *
+   * `"staticBuildModulePath": "@storybook/react/bin/build.js"`
+   */
+  staticBuildModulePath?: string;
+
+  /**
+   * The customized output dir for storybook static build.
+   * If this is empty, then it will use the storybook default output dir.
+   *
+   * @example
+   * If you want to change the static build output dir to staticBuildDir, then the static build output dir would be:
+   *
+   * `"staticBuildOutputDir": "newStaticBuildDir"`
+   */
+  staticBuildOutputDir?: string;
```

Comment:
Can you change this property name to `staticBuildOutputFolder`? We generally prefer the term "folder" and try to avoid using abbreviations in APIs.

Reply:
renamed to staticBuildOutputFolder